src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -33,6 +33,25 @@ public class AddressBookParser {
     private static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
     private static final Logger logger = LogsCenter.getLogger(AddressBookParser.class);
 
+    //NOTE DOES NOT WORK FOR TAG
+    private static enum PersonField {
+        NAME("name"),
+        PHONE("phone"),
+        EMAIL("email"),
+        ADDRESS("address"),
+        LEAD("lead"),
+        TELEGRAMHANDLE("telegramHandle"),
```

Comment:
nit: Shall we rename this to `TELEGRAM_HANDLE`?

Reply:
Good catch! this is not a nit, it could be a bug. I realised the text should be telegramhandle